pkg/bmh/baremetalhost.go

```diff
@@ -363,39 +363,24 @@ func (builder *BmhBuilder) WithOptions(options ...AdditionalOptions) *BmhBuilder
 	return builder
 }
 
-// Pull pulls existing baremetalhost from cluster.
-func Pull(apiClient *clients.Settings, name, nsname string) (*BmhBuilder, error) {
```

Comment:
Please restore the Pull function. This will be needed for many tests

Reply:
Hey I still see that pull is removed

Reply:
I restored it now, but pay attention that I have Get method